client/views/directives/previewpictures.html

```diff
@@ -0,0 +1,4 @@
+<i
+  ng-if="$ctrl.pictures.length > 0"
```

Comment:
The directive adds `click` listener, but the template conditionally renders the element - this looks like a potential for crash. Hint: you can provide function for [`template`](https://docs.angularjs.org/api/ng/service/$compile#-template-)